drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -1007,20 +1010,21 @@ protected void checkValidAndUniqueInstanceHeaderTitle(String instanceHeaderCellV
                 .filter(elem -> elem.getInformationHeaderMetaData() != null)
                 .map(ScenarioGridColumn::getInformationHeaderMetaData)
                 .anyMatch(elem -> Objects.equals(elem.getTitle(), instanceHeaderCellValue))) {
-            throw new Exception(instanceHeaderCellValue + " has already been used inside the current group");
+            throw new IllegalArgumentException(ScenarioSimulationEditorConstants.INSTANCE.
+                    instanceTitleAssignedError(instanceHeaderCellValue));
         }
     }
 
     /**
      * Verify if the given value is not already used as property header name <b>inside the same instance</b>
      * @param propertyHeaderCellValue
      * @param columnIndex
-     * @throws Exception if the given <b>propertyHeaderCellValue</b> contains a <i>dot</i> <b>OR</b> it has already been used
-     * inside the <b>instance</b> of the given column
+     * @throws IllegalArgumentException if the given <b>propertyHeaderCellValue</b> contains a <i>dot</i> <b>OR</b>
+     * it has already been used inside the <b>instance</b> of the given column <b>OR</b> the value is <i>empty</i>
      */
-    protected void checkValidAndUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) throws Exception {
+    protected void checkValidAndUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) {
```

Comment:
@yesamer 
I think now we also have to check that user does not set any "reserverd keyword" (namely, "ScesimExpression")

Reply:
@gitgabrio I'm not totally convinced this check is strictly necessary, considering this is only a label, not used in backend side. Anyway, based on your suggestion, I added the check. Now, only the "expression" column accepts the "reserved keyboard" as suffix of the property title.